css/style.css

```diff
@@ -1,6 +1,881 @@
+body {
+	margin: 0;
+	box-sizing: border-box;
+}
+
+.conteiner {
+	max-width: 1030px;
+	width: 100%;
+	margin: 0 auto;
+	box-sizing: border-box;
+
+}
+
+ a {
+ 	display: inline-block;
+ 	text-decoration: none;
+ }
+
+ .header {
+ 	width: 100%;
+ 	top: 0;
+ 	left: 0;
+ 	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 116px;
+ }
+
+.header__column {
+	flex: 0 0 55px;
+	height: 55px;
+}
+
+.header__logo {
+	flex: 0 0 55px;
+}
+
+.header__logo_img {
+	max-width: 100%;
+	display: block;
+}
+
+.header__logo_tabimg {
+	display: none;
+}
+
+.header-menu {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	width: 334px;
+	height: 55px;
+	margin: 0 145px 0 0;
+}
+
+
+.header-menu__link {
+	font-family: GothaPro;
+	font-size: 12px;
+	text-align: justify;
+	text-transform: uppercase;
+	color: #000;
+	font-weight: bold;
+}
+
+.header-menu__burger {
+	display: none;
+}
+
+.search-form__input {
+	height: 27px;
+	width: 178px;
+	border-bottom: 2px solid #000;
+	background-color: #fff;
+	display: flex;
+}	
+
+.search-form__text {
+	font-family: 'GothaPro';
+	font-size: 14px;
+	border: none;
+	outline: none;
+  	vertical-align: middle;
+  	width: 157px;	
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)